src/GeoTagFunctions.java

```diff
@@ -66,20 +69,59 @@ public static boolean readImageMeta(File jpeg) {
     /* protected ... void printGeoTagData(...) {
         ...
     } */
+    
+    //Pre: latitude and longtitude should be passed as two double value.
+    //Return: return true if geotag is successfully written. false otherwise.
+    //Output: a image with new geotag written in. If the wrriten process failed, no change would happen
+    public static boolean updateGeoTagData(File jpeg, File result, double latitude, double longtitude) {
+    	return writeGeoTagData(jpeg, result, latitude, longtitude);
+    }
```

Comment:
When removeGeoTagData() is complete, will updateGeoTagData() call removeGeoTagData() first and then writeGeoTagData()?

Because right now, updateGeoTagData() and writeGeoTagData() do the same thing

Reply:
It update geotag if the geotag exists or add geotag if there is not one. Both jobs are covered.